### PR TITLE
Add catalog docs

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

File: docs/catalog.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# What is the LEAP Data Catalog?
+
+The LEAP Data Catalog is a centralized repository that consolidates various LEAP datasets into a single JSON file. This JSON catalog is generated from individual YAML files, each representing a dataset. The catalog is currently in development, and a draft version is available at [LEAP Data Catalog](https://leap-data-catalog.vercel.app/).
+
+## The Schema
+
+The catalog is generated from individual YAML files. Each dataset, or feedstock, needs two files: `meta.yaml` and `catalog.yaml`. These files can be located in different locations to separate metadata curated during dataset creation (like Pangeo-Forge recipes) from catalog information that enhances the metadata for the LEAP catalog.
```
Comment (norlandrhagen) [inline docs/catalog.md:7]: Maybe a RW? -> ...or feedstock, needs two files. A `meta.yaml` file and  a`catalog.yaml` file.

Comment (norlandrhagen) [reply docs/catalog.md:7]: I think you can cut located. RW -> These files can be in different locations...

Comment (norlandrhagen) [reply docs/catalog.md:7]: Not super clear on what this means. Mayce we can cut? `that enhances the metadata for the LEAP catalog.`

File: docs/catalog.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+### meta.yaml Schema
+
+The `meta.yaml` schema is borrowed from the [Pangeo-Forge](https://pangeo-forge.org/) project. The following fields are required:
+
+| Field         | Type             | Description                                  | Object Properties                                                                                                                                                 |
```
Comment (norlandrhagen) [inline docs/catalog.md:13]: I find the rendered type hinting a bit cluttered, but don't have strong feelings.

image

File: docs/catalog.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+| `links`         | Array of Objects | Additional links related to the feedstock.       | `label`: Label of the link (Type: String)<br>`url`: URL of the link (Type: String)                                                                                                                                                                                                                                                                            |
+| `stores`        | Array of Objects | Information about where the feedstock is stored. | `id`: ID of the store (Type: String)<br>`name`: Name of the store (Type: String, optional)<br>`url`: URL of the store (Type: String)<br>`rechunking`: Rechunking information (Type: Array of Objects, optional)<br>`public`: Whether the store is public (Type: Boolean, optional)<br>`geospatial`: Whether the store is geospatial (Type: Boolean, optional) |
+
+### Example YAML Files
```
Comment (norlandrhagen) [inline docs/catalog.md:32]: I really like the inclusion of the example files!

File: docs/catalog.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    name: "The cool small Proto Dataset"
+    url: "gs://leap-scratch/data-library/feedstocks/proto_feedstock/small.zarr"
+    rechunking:
+      - path: "gs://some-bucket/small.zarr"
+        use_case: "multiscales"
+  - id: "large"
+    name: "The even cooler large Proto Dataset"
+    url: "gs://leap-scratch/data-library/feedstocks/proto_feedstock/large.zarr"
+```
+
```
Comment (norlandrhagen) [inline docs/catalog.md:79]: Maybe worth adding a note that the validation is ran via GH actions and if the user should rely on that or use the CLI first and foremost.

File: docs/catalog.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+✅ Valid feedstock: (1)
+  📂 https://github.com/leap-stc/proto_feedstock/blob/main/feedstock/catalog.yaml
+```
+
+### How to Add a New Dataset to the LEAP Web Catalog
+
+To add a new dataset to the LEAP web catalog, follow these steps:
+
+1. **Create YAML Files**: Create `meta.yaml` and `catalog.yaml` files for your dataset as shown in the examples above. These can reside in a GitHub repository or any other location accessible via a URL.
+2. **Add Dataset URL**: Add the URL of your dataset's `catalog.yaml` file to this [file](https://github.com/leap-stc/data-management/blob/main/catalog/input.yaml).
```
Comment (norlandrhagen) [inline docs/catalog.md:114]: Don't know if it's worth it, but might be nice to have a note that they can edit that file by clicking on the 
image icon.

**Review by norlandrhagen — APPROVED** [review]
Looks great @andersy005! A few minor comments, but take them or leave them.

**Comment by jbusecke** [comment]
This looks really good @andersy005. Left a few suggestions and generally agree with all the comments @norlandrhagen made.

**Comment by jbusecke** [comment]
Should we get this merged?

**Comment by andersy005** [comment]
> Should we get this merged?

not yet. i intend to address comments from @norlandrhagen tomorrow or thursday.

**Pull request closed without merging**